test/java/scenarios.json

```diff
@@ -3,7 +3,7 @@
         "image": "ubuntu:focal",
         "features": {
             "java": {
-                "version": "19",
```

Comment:
This fails on `main` branch as well, hence, the PR changes are unrelated. See https://github.com/devcontainers/features/actions/runs/7562535809/job/20593192063#step:4:1092

Reply:
Should I remove these changes?

Reply:
Should be fine, thank you!